translations/monero-core_de.ts

```diff
@@ -954,22 +954,22 @@
     <message>
         <location filename="../pages/Settings.qml" line="102"/>
         <source>Create view only wallet</source>
-        <translation type="unfinished"></translation>
+        <translation>Schreibgeschütze Wallet erstellen</translation>
```

Comment:
"beschränkter Lesezugriff" sounds like view-only is even more limited. "Wallet nur mit Leseberechtigung erstellen" is the best one I think.